django_mailbox/management/commands/getmail.py

```diff
@@ -4,27 +4,10 @@
 
 from django_mailbox.models import Mailbox
 
-
 logger = logging.getLogger(__name__)
```

Comment:
I know this is where _I_ put this, but could I convince you into moving these two logger lines into the `handle` method below so importing this module is side-effect free?

Reply:
done with f531c7865bfcec8d62f01af1a03563dcfd48cb57